cmd/main.go

```diff
@@ -59,8 +63,13 @@ var (
 	customBigtableInfo = flag.String("custom_bigtable_info", "", "Yaml formatted text containing information for custom Bigtable")
 	// Branch Bigtable Cache
 	useBranchBigtable = flag.Bool("use_branch_bigtable", true, "Use branch bigtable cache")
+	// SQLite database
+	useSQLite = flag.Bool("use_sqlite", false, "Use SQLite as database.")
+	sqliteDB  = flag.String("sqlite_db", "./sqlite/datacommons.db", "SQLite DB file path.")
 	// Stat-var search cache
 	useSearch = flag.Bool("use_search", true, "Uses stat var search. Will build search indexes.")
+	// Include maps client
+	useMapsApi = flag.Bool("use_maps_api", true, "Uses maps API for place recognition.")
```

Comment:
Is this for the recon API implementation?

Reply:
Yes.